extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -98,8 +98,11 @@ export class AmpAd3PImpl extends AMP.BaseElement {
 
     setupA2AListener(this.win);
 
-    /** @private {?Element|undefined} */
+    /** @private {?string|undefined} */
     this.container_ = undefined;
```

Comment:
I use `undefined` to avoid calling `getAdContainer` multiple times. Can we do
`this.container_ = this.container_ || null` to get rid of the `undefined` value?